docs/cli/discount-code-generator.md

```diff
@@ -0,0 +1,155 @@
+# Discount Code Generator
+
+Generate unique discount codes to be imported to the [commercetools platform](https://dev.commercetools.com/). See usage below
+
+## Usage
+`npm install @commercetools/discount-code-generator --global`
+
+### CLI
+```
+Usage: discount-code-gen [options]
+
+Options:
+  --help, -h                 Show help text.                                                     [boolean]
+  --version, -v              Show version number.                                                [boolean]
+  --quantity, -q             Quantity of discount codes to generate. (Between 1 and 500000)     [required]
+  --code-length, -l          Length of the discount codes to generate.                       [default: 11]
+  --code-prefix, -p          Prefix for each code. No prefix will be used if omitted.        [default: ""]
+  --input, -i                Path to code options CSV or JSON file.
+  --output, -o               Path to store generated output file.                      [default: "stdout"]
+  --delimiter, -d            Used CSV delimiter for input and/or output file.               [default: ","]
+  --multivalueDelimiter, -m  Used CSV delimiter in multivalue fields for input/output file. [default: ";"]
+  --logLevel,                Logging level: error, warn, info or verbose.                [default: "info"]
```

Comment:
Just a nitpick - some params are written in `camelCase` and some in `kebab-case`